h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -1357,43 +1357,53 @@ def show_test_results(test_name, curr_test_val, new_test_val):
         return 0
 
 
-def assert_H2OTwoDimTable_equal(table1, table2, col_header_list, tolerance=1e-6, check_sign=False):
+def assert_H2OTwoDimTable_equal(table1, table2, col_header_list, tolerance=1e-6, check_sign=False, check_all=True,
```

Comment:
See, it's not about allowing checking all of them. The approach is not good. As Guido van Rossum said, if you have a boolean parameter, you actually have two functions stuffed into one. The function generateForIndices() or something, there are actually two functions. It makes more sense to refactor assert_H2OTwoDimTable_equal and have another function, assert_H2OTwoDimTable_equal_most_probably, that checks randomly (that's another issue whether it's a good idea).
Anyway, I don't mind approving it, but we have to move to better programming practices.